examples/ephemeral-spark/tamr-vm.tf

```diff
@@ -3,7 +3,7 @@ module "tamr-vm" {
 
```

Comment:
If you wanted to close out https://datatamr.atlassian.net/browse/DEV-14726 while we're here, we could update this vm module to be `1.0.2`